python/ray/_private/structured_logging/logging_config.py

```diff
@@ -0,0 +1,58 @@
+from ray._private.structured_logging.constants import LOG_MODE_DICT
+from ray.util.annotations import PublicAPI
+
+from dataclasses import dataclass
+
+
+@PublicAPI(stability="alpha")
+@dataclass
```

Comment:
`kw_only` is added in Python 3.10. It may be incompatible with some Ray images.

<img width="1061" alt="Screenshot 2024-06-03 at 4 27 31 PM" src="https://github.com/ray-project/ray/assets/20109646/ef80276f-e346-4c87-976c-6a6dee083683">

Reply:
Got it.